sdks/python/apache_beam/transforms/core.py

```diff
@@ -1448,18 +1448,6 @@ def get_output_type(self):
     return (self.get_type_hints().simple_output_type(self.label) or
             self.infer_output_type(None))
 
-  def expand(self, pbegin):
-    from apache_beam.io import iobase
-    assert isinstance(pbegin, pvalue.PBegin)
-    self.pipeline = pbegin.pipeline
-    coder = typecoders.registry.get_coder(self.get_output_type())
-    source = self._create_source_from_iterable(self.value, coder)
```

Comment:
<!--new_thread; commit:200b55c1a9cacddf501869d3127f34e735e68f37; resolved:0-->
What is this?  "return Windowing(GlobalWindows())"